core/src/scanner.rs

```diff
@@ -564,9 +564,7 @@ const SYMBOLS: SymbolsMap = generate_map(&[
 	(
 		':',
 		SymbolType::Symbols(
-			generate_map(&[
```

Comment:
why is this changed?

Reply:
maybe it got the changes from fixing the duplication in the TokenType enum, and rustfmt formated this